src/wazuh_db/wdb.h

```diff
@@ -2365,4 +2365,26 @@ int wdb_task_get_upgrade_task_by_agent_id(wdb_t* wdb, int agent_id, char **node,
 // Finalize a statement securely
 #define wdb_finalize(x) { if (x) { sqlite3_finalize(x); x = NULL; } }
 
+/**
+ * @brief Method to parse the "wazuhdb getconfig" commands.
+ *
+ * @param config_source Where the config will be read from: "internal" or "wdb"
```

Comment:
Should this be "wdb" or i.e. "wdb section"

Reply:
Done!